src/run/run.c

```diff
@@ -1133,18 +1133,28 @@ static int transient_service_set_properties(sd_bus_message *m, const char *pty_p
         }
 
         if (pty_path) {
-                _cleanup_close_ int pty_slave = -EBADF;
+                if (arg_transport == BUS_TRANSPORT_MACHINE)
+                        /* Temporary work-around for https://github.com/systemd/systemd/pull/33216. */
```

Comment:
We use // for FIXME style commits.

But frankly, why not solve this properly. THis would be nicely testable, hence I see that as limited risk

Reply:
We discussed that a new API call should be added. We *really* don't want to add new APIs or do any other visible changes at this point.